openedx_events/analytics/data.py

```diff
@@ -0,0 +1,31 @@
+"""
+Data attributes for events within the architecture subdomain ``analytics``.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+
+The attributes for the events come from the CourseDetailView in the LMS, with some unused fields removed
+(see deprecation proposal at https://github.com/openedx/public-engineering/issues/160)
+"""
+
+from datetime import datetime
+
+import attr
+
+
+@attr.s(frozen=True)
+class TrackingLogData:
+    """
+    Data describing tracking log data.
+
+    Arguments:
+        name (str): course name
+        timestamp (datetime): course start date
+        data (dict): dictionary of extra data (optional)
+        context (dict): dictionary of context data.
```

Comment:
Please, update the docstring according to the implementation. If it's already up to date, try adding a bit more info